tests/api-watch-diff/test.ml

```diff
@@ -2,3 +2,214 @@ let%expect_test "test_diff_interface" =
   let result = Api_watch_diff.diff_interface ~reference:[] ~current:[] in
   Format.printf "%b" result;
   [%expect {|false|}]
+
+open Types
+
+(* Signature for type t:
+     > type t = int *)
+
+let t_sig =
+  Sig_type
+    ( Ident.create_persistent "t",
+      {
+        type_params = [];
+        type_arity = 0;
+        type_kind = Type_abstract;
+        type_private = Public;
+        type_manifest =
+          Some
+            (Transient_expr.type_expr
```

Comment:
I think you can use `Typing.create_expr` directly here instead of `Transient_expr. (type_exr (create ...))`.